flask_rest_api/utils.py

```diff
@@ -32,19 +32,23 @@ def get_appcontext():
     return ctx.flask_rest_api
 
 
-def load_info_from_docstring(docstring):
+def load_info_from_docstring(docstring, sep="---"):
     """Load summary and description from docstring"""
     split_lines = trim_docstring(docstring).split('\n')
 
-    # Info is separated from rest of docstring by a '---' line
-    for index, line in enumerate(split_lines):
-        if line.lstrip().startswith('---'):
-            cut_at = index
-            break
-    else:
-        cut_at = index + 1
+    # Default case: sep is None, thus no 'info' lines to split out.
```

Comment:
None is not the default case.

Reply:
Yeah, it's poorly worded. I actually mean something more like "assume that a separator is not found". I'll update it.